editor/src/components/text-editor/text-handling.ts

```diff
@@ -266,5 +265,5 @@ export function fixedSizeDimensionHandlingText(
 ): number {
   // Fixed dimensions for a text containing element need to be rounded up to prevent wrapping.
   const containsText = MetadataUtils.targetTextEditableAndHasText(metadata, pathTrees, elementPath)
-  return containsText ? roundUpToNearestHalf(dimensionValue) : dimensionValue
+  return containsText ? Math.ceil(dimensionValue) : dimensionValue
```

Comment:
should this be `roundUpToNearestWhole`? 🤔

Reply:
I was going to add that, but since that was just an alias for `Math.ceil` I didn't see the point in the end

Reply:
I guess the defensive take to prefer `roundUpToNearestHalf` here would be that this is kind of buried and this whole sub-integer pixel fight doesn't sound like game over

Reply:
So the reason for this rounding in the first place is so that when hugging text we don't ever round the pixel value down, as that would cause the text to wrap

Reply:
> So the reason for this rounding in the first place is so that when hugging text we don't ever round the pixel value down, as that would cause the text to wrap

that feels like it should be something prominently visible in code, whether through a comment or a named function call. In six months, I could totally see someone refactoring this while trying to achieve something totally text-unrelated... and this capability accidentally goes the way of the dodo. Unless there's a test that'd break ofc.

Reply:
There's a comment 2 lines above this. I just checked and the test doesn't exist though, so I can totally add that

Reply:
Ok I've just been writing a test for this, and it turns out that actually this functionality has been broken for a while. That's because wherever this function is called, we're using measured values that have already been rounded.